test.py

```diff
@@ -0,0 +1,9 @@
+import requests
+
+url = 'http://localhost:8000/login'
+username = raw_input('username: ')
```

Comment:
there shouldn't be any code out side of a function, you should use python unittest module for testing